packages/core/src/components/DataTable/ColumnLabels.tsx

```diff
@@ -48,12 +47,11 @@ export default function ColumnLabels({
   rowHeight?: RowHeightOptions;
   columnHeaderHeight?: HeightOptions;
   expandable?: boolean;
-  selectable?: boolean;
   columnMetadata?: ColumnMetadata;
   columnLabelCase?: ColumnLabelCase;
 }) {
   return ({ className, columns, style }: ColumnLabelsProps) => {
-    const leftmostIdx = Number(expandable) + Number(selectable);
+    const leftmostIdx = Number(expandable);
```

Comment:
I would make it explicit `const leftMostIdx = expandable ? 1 : 0` if the addition magic is not needed anymore.